docs/examples/test-app/test_crd.yaml

```diff
@@ -0,0 +1,56 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.4
+  creationTimestamp: null
+  name: testcrds.test.crd.com
+spec:
+  group: test.crd.com
+  names:
+    kind: TestCRD
+    listKind: TestCRDList
+    plural: testcrds
+    singular: testcrd
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      description: TestCRD is the Schema for the testcrds API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: TestCRDSpec defines the desired state of TestCRD
+          properties:
+            foo:
+              description: Foo is an example field of TestCRD. Edit TestCRD_types.go
+                to remove/update
+              type: string
+          type: object
+        status:
+          description: TestCRDStatus defines the observed state of TestCRD
+          type: object
+      type: object
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
```

Comment:
Could we add a condition struct array here ? 

Similar to 
https://github.com/kubernetes-sigs/application/blob/master/config/crd/bases/app.k8s.io_applications.yaml#L445

Reply:
This test includes CRD only without the controller, so the conditions wouldn't be updated and it makes no difference if we have a condition struct array or not. For simplicity, I would keep it empty.